assets/js/index.js

```diff
@@ -0,0 +1,458 @@
+  const li6 = document.createElement("li");
+  li6.setAttribute("class", "results-item");
+  li6.textContent =
+    "question: " +
+    tableArr[5].question +
+    "\n" +
+    "response: " +
+    tableArr[5].value;
+  const li7 = document.createElement("li");
+  li7.setAttribute("class", "results-item");
+  li7.textContent =
+    "question: " +
+    tableArr[6].question +
+    "\n" +
+    "response: " +
+    tableArr[6].value;
+  const li8 = document.createElement("li");
+  li8.setAttribute("class", "results-item");
+  li8.textContent =
+    "question: " + tableArr[7].question + "response: " + tableArr[7].value;
+
+  ul.append(li1, li2, li3, li4, li5, li6, li7, li8);
+
+  div.append(ul);
+  section.append(title, h2, timer, div);
+  main.append(section);
+};
+//function to render form
+const renderForm = () => {
+  const section = document.createElement("section");
+  section.setAttribute("class", "form-section content-section");
+  section.setAttribute("id", "form-container");
+  const h2 = document.createElement("h2");
+  h2.setAttribute("class", "content-section-title");
+  h2.textContent = "submit your feedback";
+  const form = document.createElement("form");
+  form.setAttribute("class", "form-section");
+
+  const inputDiv = document.createElement("div");
+  inputDiv.setAttribute("class", "form-control");
+  const input = document.createElement("input");
+  input.setAttribute("class", "form-input");
+  input.setAttribute("type", "text");
+  input.setAttribute("id", "full-name");
+  input.setAttribute("placeholder", "enter full name");
+
+  inputDiv.append(input);
+  const submitDiv = document.createElement("div");
+  submitDiv.setAttribute("class", "form-control");
+  const button = document.createElement("button");
+  button.setAttribute("class", "btn");
+  button.setAttribute("type", "submit");
+  button.textContent = "submit";
+  submitDiv.append(button);
+  form.append(inputDiv, submitDiv);
+  section.append(h2, form);
+  main.append(section);
+
+  // add event listener to form submission
+  form.addEventListener("submit", handleFormSubmit);
+};
+//function to render question
+const renderQuestion = () => {
+  //get current question
+  const currentQuestion = questions[questionIndex];
+  // create section
+  const section = document.createElement("section");
+  section.setAttribute("class", "content-section question-section");
+  section.setAttribute("id", "question-container");
+  // creat h2
+  const h2 = document.createElement("h2");
+  h2.setAttribute(
+    "class",
+    "question-section-title animate__animated animate__bounceInDown"
+  );
+  // specific to the question (dynamic)
+  h2.textContent = `${questionIndex + 1}. ${currentQuestion.text}`;
+  // creat ul and append 4 li
+  const ul = document.createElement("ul");
+  ul.setAttribute(
+    "class",
+    "feedback-list animate__animated animate__bounceInDown"
+  );
+
+  // call function to create and append list items to ul****
+  const li1 = document.createElement("li");
```

Comment:
Similarly to the code which produces your results table, this is not the ideal way to produce the elements for your answers. Since you need to reproduce the same lines of code over and over again, we should be using some form of loop instead (either a `for` loop or a `.map()`